src/Core/Mesh/MeshTypes.hpp

```diff
@@ -13,11 +14,9 @@ namespace Ra
         typedef Vector2ui Edge;
         typedef Vector3ui Triangle;
 
-        typedef uint TriangleIdx;
-        typedef uint VertexIdx;
-        typedef uint HalfEdgeIdx;
-
-        enum { InvalidIdx = uint( -1 ) };
+        using TriangleIdx = Index;

```

Comment:
why `using` for these and `typedef` for the two above?

Reply:
The question could be more general. Do we have to use the (old-style) typedef or the more C++ clean "using" for type aliasing ?
I prefer the second ... I canchange all the typedef to using if tis is OK ...

Reply:
Yes, please.